types/plotly.js/index.d.ts

```diff
@@ -856,6 +857,16 @@ export interface Margin {
     pad: number;
 }
 
+export interface ModeBar {
+    activecolor: Color;
+    add: ModeBarDefaultButtons[];
+    bgcolor: Color;
+    color: Color;
+    orientation: 'v' | 'h';
+    remove: ModeBarDefaultButtons[];
```

Comment:
similar:
```suggestion
    remove: ModeBarDefaultButtons | ModeBarDefaultButtons[];
```
`Type: string or array of strings`